### DISCUSSION TITLE
Sharing scripts between workspaces?

### DISCUSSION THREAD

**Discussion by tennox** [windmill-labs/windmill#4986] [Questions] [answered]
Hey there :)
I like to separate some things into workspaces, but I've come to realize I cannot import scripts from the other workspace. That makes it rather inconvenient for me, as there will be some scripts (e.g. that are not on the wmill hub yet) that I will want to use in all workspaces.

Is there a way?

**Comment by rubenfiszel** [comment, marked as answer]
The expected way would be to use the EE private hub https://www.windmill.dev/docs/core_concepts/private_hub

**Reply by tennox** [reply]
So, the public hub would be the way for normies like us?

**Reply by rubenfiszel** [reply]
That or do automation to sync common scripts to all desired workspaces using the cli
